pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -0,0 +1,17 @@
+package management
+
+import (
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/types/config"
+)
+
+func addDefaultPodSecurityAdmissionConfigurationTemplates(management *config.ManagementContext) error {
+	psapts := management.Management.PodSecurityAdmissionConfigurationTemplates("")
+	if _, err := psapts.Create(v3.NewPodSecurityAdmissionConfigurationTemplateRestricted()); err != nil {
+		return err
```

Comment:
I believe we need to ignore "already exists" errors here, see https://github.com/rancher/rancher/blob/68215bcdde090854ff28b43b01d1aa0b69611920/pkg/data/management/podsecuritypolicytemplate_data.go#L149

Otherwise, we will have these errors bubble up on every Rancher leader change.

Also it's helpful to wrap this error before returning, similar how it's at the linked code (with `%w` instead of `%v` though).